runtime/Cpp/CMakeLists.txt

```diff
@@ -222,14 +223,14 @@ endif(ANTLR4_INSTALL)
 
 if(EXISTS LICENSE.txt)
 install(FILES LICENSE.txt
-        DESTINATION "share/doc/libantlr4")
+        DESTINATION ${CMAKE_INSTALL_DOCDIR})
```

Comment:
Shouldn't this use an ANTLR-specific sub-directory? :thinking:

Reply:
`CMAKE_INSTALL_DOCDIR` contains a project specific sub-directory by default, see https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html#result-variables. It defaults to `DATAROOTDIR/doc/PROJECT_NAME`.

Reply:
Oh, interesting. Didn't know that.

I wonder whether changing the casing of the project name has other side effects though :thinking: